src/status_im/keycard/change_pin.cljs

```diff
@@ -96,23 +96,27 @@
   {:events [:keycard.callback/on-change-pin-error]}
   [{:keys [db] :as cofx} error]
   (log/debug "[keycard] change pin error" error)
-  (let [tag-was-lost? (= "Tag was lost." (:error error))
-        pairing       (common/get-pairing db)]
+  (let [tag-was-lost? (common/tag-lost? (:error error))
+        pairing       (common/get-pairing db)
+        pin-retries (common/pin-retries (:error error))]
     (fx/merge cofx
               (if tag-was-lost?
                 (fx/merge cofx
                           {:db (assoc-in db [:keycard :pin :status] nil)}
                           (common/set-on-card-connected :keycard/change-pin))
-                (if (re-matches common/pin-mismatch-error (:error error))
+                (if (not (nil? pin-retries))
```

Comment:
nitpicking, but `(if-not foo)` is preferable over `(if (not foo))` https://github.com/bbatsov/clojure-style-guide#if-not

Reply:
thanks, fixed